.github/workflows/test.yaml

```diff
@@ -278,10 +278,10 @@ jobs:
       run: |
           # Register a problem matcher to highlight spread failures
           echo "::add-matcher::.github/spread-problem-matcher.json"
-          export SPREAD_BUILD_SNAPD_FROM_CURRENT=true
-          export SPREAD_ENABLE_KVM=true
+          export SPREAD_NESTED_BUILD_SNAPD_FROM_CURRENT=true
```

Comment:
My point about naming was that `SPREAD_` is not our namespace, this should be called `NESTED_BUILD_SNAPD_FROM_CURRENT`. This comment applies to all the variables called `SPREAD_NESTED`.

Reply:
done